src/gcinfo/gcinfoencoder.cpp

```diff
@@ -1217,7 +1217,7 @@ void GcInfoEncoder::Build()
         // (after, of course, adding the size of the call instruction to get the return PC).
         callSite += m_pCallSiteSizes[callSiteIndex] - 1;
 
-        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite)) == callSite);
+        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite + 1)) == (callSite + 1));
```

Comment:
@JonghyunPark1302, I don't think this change is right. We're normalizing the callsite-offset in the next line. We should test that we can denotmalize back the normalized thing correctly.